packages/entities/entities-gateway-services/src/components/GatewayServiceForm.vue

```diff
@@ -714,7 +714,7 @@ const saveTlsVerify = (gatewayService: Record<string, any>) => {
   delete gatewayService.tls_verify_value
 }
 
-const getPayload = computed((): Record<string, any> => {
+const getPayload = (): Record<string, any> => {
```

Comment:
This function is `computed` in every other instance in the repo - why would it be changed only here?

Also, it is exposed via `defineExpose` in this component - does the usage throughout the repo where it is used need to be updated?

Reply:
It was a static object earlier so I'm basically reverting my change.

Reply:
Only in this one instance?